qml/components/MessageListViewItem.qml

```diff
@@ -136,12 +137,12 @@ ListItem {
             messageBackground.isUnread = index > chatModel.getLastReadMessageIndex();
         }
         onLastReadSentMessageUpdated: {
-            console.log("[ChatModel] Messages in this chat were read, new last read: " + lastReadSentIndex + ", updating description for index " + index + ", status: " + (index <= lastReadSentIndex));
+            Debug.log("[ChatModel] Messages in this chat were read, new last read: " + lastReadSentIndex + ", updating description for index " + index + ", status: " + (index <= lastReadSentIndex));
             messageDateText.text = getMessageStatusText(myMessage, index, lastReadSentIndex, messageDateText.useElapsed);
         }
         onMessageUpdated: {
             if (index === modelIndex) {
-                console.log("[ChatModel] This message was updated, index " + index + ", updating content...");
+                Debug.log("[ChatModel] This message was updated, index " + index + ", updating content...");
```

Comment:
Here too... and there are many more places where strings are being allocated and formatted only to be thrown away if debug output is disabled.